seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -454,16 +453,14 @@
       "challengeSeed": [
         "let a = 8, b = 6;",
         "/* Alter code below this line */",
-        "a = b;",
-        "b = a;",
         "/* Alter code above this line */",
         "console.log(a); // should be 6",
         "console.log(b); // should be 8"
       ],
       "tests": [
-        "// Test a is 6",
-        "// Test b is 8",
-        "// Test destructuring was used"
+        "assert(a === 6, 'message: Value of <code>a</code> should be 6, after swapping.');",
+        "assert(b === 8, 'message: Value of <code>b</code> should be 8, after swapping.');",
+        "assert(/\\[\\s*.{1}\\s*,\\s*.{1}\\s*\\];/g.test(code), 'message: Use array destructuring to swap a and b.');"
```

Comment:
@Greenheart Update - 

I tried out asserting both sides of the expression in 2 different ways - 

1. First one was your idea, with capture groups and backreferencing - 
`"assert(/\\[\\s*(.)\\s*,\\s*(.)\\s*\\]\\s*=\\s*\\[\\s*\\2\\s*,\\s*\\1\\s*\\]/g.test(code), 'message: Use array destructuring to swap a and b.');"`
2. Second one was using a `.{1}` to match any character - 
`"assert(/\\[\\s*.{1}\\s*,\\s*.{1}\\s*\\]\\s*=\\s*\\[\\s*.{1}\\s*,\\s*.{1}\\s*\\]/g.test(code), 'message: Use array destructuring to swap a and b.');"`

None of the two ways worked in my local instance, while both worked in regexr - 

![screenshot 2017-02-20 18 02 49](https://cloud.githubusercontent.com/assets/11348778/23144762/dc897692-f797-11e6-9cbd-efb6cf7f6109.png)
![screenshot 2017-02-20 18 03 05](https://cloud.githubusercontent.com/assets/11348778/23144764/e3911490-f797-11e6-8d33-fe642033a9a3.png)

I don't know for sure but it might be due to transpiling issues as Timo mentioned in the chatroom.

I can either comment out the test entirely or perhaps we could keep the test in my original PR. I think having *some* form of assertion/validation is better than none (for now).

Your thoughts?